wasmcloud-test-util/src/provider_test.rs

```diff
@@ -317,21 +292,26 @@ pub async fn start_provider_test(config: TomlMap) -> Result<Provider, anyhow::Er
 
     let keys = wascap::prelude::KeyPair::new_user();
     let mut host_data = HostData::default();
-    host_data.host_id = "_TEST_".to_string();
+    host_data.host_id = ld.actor_id.clone();
```

Comment:
shouldn't we at least fake a proper host ID here, e.g. `Nxxxx` ?

Reply:
I don't want to break anything now so I added a comment reminder to revisit this later.